product_yahoo_auction_sst/models/product_template.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    title = fields.Char(
+        string='Title',
+    )
+    product_category = fields.Char(
+        string='Product Category',
+    )
+    product_condition_comment = fields.Text(
+        string='Product Condition Comment',
+    )
+    accessories = fields.Char(
+        string='Accessories',
+    )
+    remark = fields.Text(
+        string='Remark',
+    )
+    staff_in_charge = fields.Many2one(
+        'hr.employee',
+        string='Staff in charge',
+    )
+    auction_start_price = fields.Monetary(
+        string='Auction Starting Price',
+    )
+    auction_buyout_price = fields.Monetary(
+        string='Auction Buyout Price',
+    )
+    product_condition = fields.Selection(
+        [('new', 'New'), ('used', 'Used')],
+        string='Product Condition',
+    )
+    delivery_method = fields.Many2one(
+        'delivery.carrier',
+        string='Delivery Method',
+    )
+    delivery_size = fields.Many2one(
```

Comment:
`carrier_size_id`

Note that the name of a many2one field should always ends with `_id`.